install/installer/pkg/components/blobserve/configmap.go

```diff
@@ -49,21 +49,13 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 						Replacement: ctx.Config.Domain,
 						Path:        "/ide/out/vs/workbench/workbench.web.main.js",
 					}, {
-						Search:      "open-vsx.org",
+						Search:      "https://open-vsx.org",
 						Replacement: openVSXProxyUrl,
 						Path:        "/ide/out/vs/workbench/workbench.web.main.js",
-					}, {
-						Search:      "{{extensionsGalleryItemUrl}}",
-						Replacement: extensionsGalleryItemUrl,
-						Path:        "/ide/out/vs/workbench/workbench.web.api.js",
 					}, {
 						Search:      "{{extensionsGalleryItemUrl}}",
 						Replacement: extensionsGalleryItemUrl,
 						Path:        "/ide/out/vs/workbench/workbench.web.main.js",
-					}, {
-						Search:      "{{trustedDomain}}",
-						Replacement: trustedDomain,
-						Path:        "/ide/out/vs/workbench/workbench.web.api.js",
```

Comment:
`workbench.web.api.js` file does not exist a long time ago so it's fine to delete these rules